### PR TITLE
Add support for SASL/TLS for ccloud

### PR DESCRIPTION
**Pull request by MacTynow**

### CODE REVIEW

File: messaging/kafka-sarama/base.go [hunk 1]
```diff
@@ -106,12 +111,34 @@ func New(option *Option) (messaging.QueueV2, error) {
...
+		}
+
+		config.Net.SASL.Password = l.Option.SaslPassword
+		if config.Net.SASL.Password == "" {
+			return nil, errors.Errorf("CCLOUD_PASSWORD not set")
+		}
+
+		config.Net.TLS.Enable = true
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: true,
```
Comment (niravgandhi) [inline messaging/kafka-sarama/base.go]: Why is this property true ?

File: messaging/kafka-sarama/base.go [hunk 2]
```diff
@@ -106,12 +111,34 @@ func New(option *Option) (messaging.QueueV2, error) {
...
 func (l *Kafka) NewListener(option *Option) (*cluster.Consumer, error) {
 	config := cluster.NewConfig()
+
 	config.Consumer.Return.Errors = true
-	config.Consumer.MaxWaitTime = l.Option.MaxWait
+	// config.Consumer.MaxWaitTime = l.Option.MaxWait
 	config.Group.Return.Notifications = true
-	config.Group.PartitionStrategy = l.Option.Strategy
-	config.Group.Heartbeat.Interval = time.Duration(l.Option.Heartbeat) * time.Second
+	// config.Group.PartitionStrategy = l.Option.Strategy
```
Comment (niravgandhi) [inline messaging/kafka-sarama/base.go]: Can you mention why we are removing these ?

Comment (MacTynow) [reply messaging/kafka-sarama/base.go]: I commented them for testing and forgot to uncomment 🤦

File: messaging/kafka-sarama/base.go (hunk 1 above)
Comment (MacTynow) [reply messaging/kafka-sarama/base.go]: Indeed, not needed.

**Pull request merged**